### PR TITLE
Added Learning thread formulae and equations we need to know

### PR DESCRIPTION
**Pull request by stellarpower**
Open for any corrections but I really need to concentrate on past papers now so I personally won't be able to make lots of changes.

### CODE REVIEW

File: _sections/inf2/inf2b.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ links:
 ---
 
 - Revision Hive Mind doc: [CLICK ME](https://drive.google.com/folderview?id=0BzxEStTA9O-SamF2YWFWRFJCNGc&usp=sharing)
+- [Revision Formulae for Learning Thread (pdf)](../../resources/inf2b/Inf2B%20Formulae.pdf) [(LaTex)](../../resources/inf2b/Inf2B%20Formulae.tex)
```
Comment (neanias) [inline _sections/inf2/inf2b.md]: You don't need the `../..` here, you can use `/resources/...`.

Comment (neanias) [reply _sections/inf2/inf2b.md]: I'd also maybe recommend removing the spaces in your file paths, or changing them to `_`.

Comment (stellarpower) [reply _sections/inf2/inf2b.md]: Will it mess things up if I leave it as is?

Comment (neanias) [reply _sections/inf2/inf2b.md]: For which point?

Comment (stellarpower) [reply _sections/inf2/inf2b.md]: The pathnames? Can I just change them or will it require reuploading and another pull request?

Comment (neanias) [reply _sections/inf2/inf2b.md]: You can change them on GitHub and it’ll add it to your pull request

**Pull request merged**